src/erlfmt_scan.erl

```diff
@@ -185,6 +191,8 @@ split_shebang(Tokens) ->
     {unicode:characters_to_list(Shebang), Rest}.
 
 %% TODO: make smarter
+stringify_tokens([]) ->
+    {"", undefined};
```

Comment:
This seems wrong, maybe it is time to make it smarter.
@michalmuskala do you have any ideas?